packages/datadog-plugin-google-cloud-pubsub/src/index.js

```diff
@@ -1,116 +1,101 @@
 'use strict'
 
+const Plugin = require('../../dd-trace/src/plugins/plugin')
+const { storage } = require('../../datadog-core')
 const analyticsSampler = require('../../dd-trace/src/analytics_sampler')
+const messageSpans = require('../../datadog-instrumentations/src/google-cloud-pubsub')
```

Comment:
The instrumentation and the plugin should be decoupled and never depend on each other.